snap/snapcraft.yaml

```diff
@@ -170,6 +170,7 @@ parts:
     stage-packages:
       - aufs-tools
       - git
+      - libcurl3-gnutls
       - libltdl7
       - pigz
       - xz-utils
```

Comment:
Unsure wich one but I bet the problem is either 2 packages are conflicting with eachother at the symlink level or is missing libssl-dev